src/components/AssetDetailsPanel/stories/helper.tsx

```diff
@@ -0,0 +1,38 @@
+import { Asset } from '@cognite/sdk';
+import { action } from '@storybook/addon-actions';
+import React from 'react';
+import {
+  ASSET_LIST_CHILD,
+  ASSET_ZERO_DEPTH_ARRAY,
+} from '../../../mocks/assetsListV2';
+import { MockCogniteClient } from '../../../utils/mockSdk';
+import { ClientSDKProvider } from '../../ClientSDKProvider';
+
+class CogniteClient extends MockCogniteClient {
+  assets: any = {
+    retrieve: (ids: { id: number }[]) =>
+      new Promise(resolve => {
+        setTimeout(() => {
+          const allAssets = [...ASSET_ZERO_DEPTH_ARRAY, ...ASSET_LIST_CHILD];
+          const result = ids.map(({ id }) => {
+            return allAssets.find(asset => asset.id === id);
+          });
+
+          resolve(result || []);
+        }, 200);
+      }),
+  };
+}
+
+const client = new CogniteClient({ appId: 'gearbox test' });
+
+export const decorators = [
+  (storyFn: any) => (
+    <ClientSDKProvider client={client}>{storyFn()}</ClientSDKProvider>
+  ),
+];
+
+export const onAssetLoaded = (asset: Asset) =>{
+  console.log(asset)
+action('handleAssetLoaded')(asset)
```

Comment:
we will not see the result of this action cause action tab is unavailable for Docs view

Reply:
Roman action tab unavailable for docs but it is shown in the canvas tab.